### ISSUE TITLE
Merge doesn't add aliased actions

### ISSUE THREAD

**Issue by afdev82** [CanCanCommunity/cancancan#280]
I use the AlchemyCMS gem to provide the CMS functionality for a rails 4.2+ app.
I have a problem since I have to merge the permissions in the CMS with permissions defined in my app.
I already described what happens in [this issue](https://github.com/AlchemyCMS/alchemy_cms/issues/739).
It seems that the merge method of the Ability object doesn't merge the aliased actions.
Is this an intentional behaviour or is it a bug?

**Comment by Haniyya** [comment]
Aliased actions are not copied over. Merge just copies rules:
```rb
    def merge(ability)
      ability.rules.each do |rule|
        add_rule(rule.dup)
      end
      self
    end
```

I agree though that this (along with a few  other things) would improve `#merge` and make it more intuitive.

**Comment by coorasse** [comment]
Closed because of starvation. 
Please feel free to reopen if this issue is still valid or interested in solving with a PR.
Thank you.

**Comment by coorasse** [comment]
duplicated of #468. Fixed by #538.
